cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -322,9 +322,6 @@ def certificate_activation_handler(request, course_key_string):
     POST
         json: is_active. update the activation state of certificate
     """
-    # Only global staff (PMs) are able to activate/deactivate certificate configuration
```

Comment:
Ah, that makes sense -- yeah as long as there's some sort of access umbrella going on here I'm good with the change.